### PR TITLE
refactor: Compact ticket non inspectable and accessibility fixes

### PR DESCRIPTION
**Pull request by jorelosorio**
Closes: https://github.com/AtB-AS/kundevendt/issues/1880

### CODE REVIEW

File: src/screens/Ticketing/Ticket/CompactTicketInfo.tsx [hunk 1]
```diff
@@ -148,9 +165,21 @@ export const getTicketInfoTexts = (
     conjunction,
     serialComma: false,
   });
-  const timeUntilExpire = t(TicketTexts.validityHeader.valid(durationText));
-
-  var accessibilityLabel = timeUntilExpire + screenReaderPause;
+  var timeUntilExpireOrInvalid: string;
+
+  if (isMobileToken(inspectableToken) && !isInspectable) {
```
Comment (gorandalum) [inline src/screens/Ticketing/Ticket/CompactTicketInfo.tsx:170]: This isn't ment to be showed if t:card is the inspectable token? 🤔

Comment (jorelosorio) [reply src/screens/Ticketing/Ticket/CompactTicketInfo.tsx:170]: The accessibility changes when the token is valid or inspectable on another device. These conditions evaluate that, but do you see something is missing? Please let me know the logic about tickets is a quite complex to follow 😅

Comment (gorandalum) [reply src/screens/Ticketing/Ticket/CompactTicketInfo.tsx:170]: It was not regarding accessibility label, but regarding the `timeUntilExpireOrInvalid` value. By just reading it, it seems that if the inspectable token is a t:card, then the normal validity header "Expires in ..." is shown. I don't know if this is intended or not.

Comment (jorelosorio) [reply src/screens/Ticketing/Ticket/CompactTicketInfo.tsx:170]: Hey! After our talk I made improvements in the overall code, now it looks slightly better and fixed with the invalid ticket!

**Review by gorandalum — APPROVED** [review]

**Pull request merged**